lib/spack/docs/packaging_tutorial/0.package.py

```diff
@@ -0,0 +1,64 @@
+# flake8: noqa
```

Comment:
Why do all of these packages need `# flake8: noqa`? Do they fail? If so we might want to fix that in `spack create`.